webpack.common.js

```diff
@@ -386,84 +346,113 @@ module.exports = {
     'socket.io-client': 'io',
     'jquery': '$',
     'moment': 'moment',
-    'handlebars': 'Handlebars',
     'highlight.js': 'hljs',
     'select2': 'select2'
   },
 
   module: {
-    rules: [{
-      test: /\.js$/,
-      use: [{ loader: 'babel-loader' }],
-      exclude: [/node_modules/, /public\/vendor/]
-    }, {
-      test: /\.css$/,
-      use: [
-        {
-          loader: MiniCssExtractPlugin.loader,
+    rules: [
+      {
+        test: require.resolve(path.join(__dirname, 'public/vendor/codemirror-spell-checker/spell-checker.min.js')),
+        use: {
+          loader: 'imports-loader',
           options: {
-            publicPath: '',
+            imports: ['default codemirror CodeMirror']
           }
-        },
-        'css-loader'
-      ]
-    }, {
-      test: /\.less$/,
-      use: [
-        MiniCssExtractPlugin.loader,
-        {
-          loader: 'css-loader',
+        }
+      },
+      {
+        test: require.resolve(path.join(__dirname, 'public/vendor/ot/ot.min.js')),
+        use: {
+          loader: 'exports-loader',
           options: {
-            importLoaders: 1
+            type: 'commonjs',
+            exports: ['ot']
           }
         },
-        'less-loader'
-      ]
-    }, {
-      test: require.resolve('js-sequence-diagrams'),
-      use: [{
-        loader: 'imports-loader',
-        options: {
-          imports: ['default lodash _', 'default raphael Raphael', 'default eve eve']
+        parser: {
+          amd: false
         }
+      },
+      {
+        // Idle.js must be imported as CommonJS, as its AMD export is broken
+        test: require.resolve('Idle.Js'),
+        parser: {
+          amd: false
+        }
+      },
+      {
+        test: /\.js$/,
+        use: [{ loader: 'babel-loader' }],
+        exclude: [/node_modules/, /public\/vendor/]
+      }, {
+        test: /\.css$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              publicPath: '',
+            }
+          },
+          'css-loader'
+        ]
+      }, {
+        test: /\.less$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1
+            }
+          },
+          'less-loader'
```

Comment:
😱 why less?

Reply:
We used that before :man_shrugging:

Reply:
Do we even have less code?...

Reply:
Bootstrap and fork-awesome seem to use it

Reply:
Okay.